data_steward/tools/import_rdr_omop.py

```diff
@@ -120,12 +120,12 @@ def create_rdr_tables(client, rdr_dataset, bucket):
         except NotFound:
             LOGGER.info(f'{table} not provided by RDR team.  Will not exist in '
                         f'rdr dataset: `{rdr_dataset}`')
-            if table in AOU_REQUIRED:
-                LOGGER.info(f'Creating empty AOU_REQUIRED table, `{table}`')
-                dest_table = f'{client.project}.{rdr_dataset}.{table}'
-                dest_table = bigquery.Table(dest_table, schema=schema_list)
-                dest_table = client.create_table(dest_table)
-                LOGGER.info(f'Created empty table `{dest_table.full_table_id}`')
+
+            LOGGER.info(f'Creating empty CDM table, `{table}`')
+            dest_table = f'{client.project}.{rdr_dataset}.{table}'
+            dest_table = bigquery.Table(dest_table, schema=schema_list)
+            dest_table = client.create_table(dest_table)
+            LOGGER.info(f'Created empty table `{dest_table.full_table_id}`')
         else:
             destination_table = client.get_table(
                 table_id)  # Make an API request.
```

Comment:
nit: I think `table_id` can be reused here. Also the same info log could apply following the try-except-else block (note un-indent).
```suggestion
            destination_table = bigquery.Table(table_id, schema=schema_list)
            destination_table = client.create_table(destination_table)
        else:
            destination_table = client.get_table(
                table_id)  # Make an API request.
        LOGGER.info(f'Loaded {destination_table.num_rows} rows into '
                    f'`{destination_table.full_table_id}`.')
```